compare-strings.js

```diff
@@ -45,8 +45,9 @@ function findBestMatch(mainString, targetStrings) {
   if (!areArgsValid(mainString, targetStrings)) {
     throw new Error('Bad arguments: First argument should be a string, second should be an array of strings');
   }
-  var ratings = _.map(targetStrings, function (targetString) {
+  var ratings = _.map(targetStrings, function (targetString, i) {
```

Comment:
Ratting is used to define minimum accurate in the search
ex:
```
if(search['bestMatch']['rating'] < 0.2){

}
```
Returning index of the element, possibility use this script to search element in array and obtain your position.